INSTALL.md

```diff
@@ -135,27 +155,28 @@ To learn more about the release, try:
   $ helm status nydus-snapshotter
   $ helm get all nydus-snapshotter
 ```
-# 3. Run Nydus image
-## 3.1 Wait for Nydus components to be ready
-```
-pod=`kubectl -n nydus-snapshotter get pods --no-headers -o custom-columns=NAME:metadata.name`
-echo "snapshotter pod name ${pod}"
-# Wait for Nydus components to be ready
-kubectl -n nydus-snapshotter wait po $pod --for=condition=ready --timeout=2m
-```
-The process may wait for about ten seconds.
-``` bash
-root@iZj6c6r36jb1qak6ajdw4oZ:~/test/helm# kubectl -n nydus-snapshotter wait po $pod --for=condition=ready --timeout=2m
-snapshotter pod name nydus-snapshotter-bkmkr
-pod/nydus-snapshotter-bkmkr condition met
-```
-## 3.2 Start the pod based on the Nydus image
+
+Check that Nydus is deployed successfully:
+
+```shell
+$ kubectl wait po --all -n nydus-snapshotter --for=condition=ready --timeout=1m
+pod/nydus-snapshotter-6mwlv condition met
 ```
-cat <<EOF > nydus-pod.yaml
+
+<!-- markdownlint-restore -->
+
+## Run Nydus Image in Kubernetes
+
+Create Nginx configuration file `nginx-nydus.yaml` and
```

Comment:
Create nginx pod configuration file `nginx-nydus.yaml` with nydus image `ghcr.io/dragonflyoss/image-service/nginx:nydus-latest`.

For more details about how to build nydus image, please refer to the [document](https://github.com/containerd/nerdctl/blob/master/docs/nydus.md#build-nydus-image-using-nerdctl-image-convert).